README.md

```diff
@@ -1,568 +1,181 @@
+## Why is this important?
 
-For example, you can use aiconfig to create a chatbot, and use the same format to save the chat history so it can be resumed for the next session.
+Today application code is tightly coupled with the gen AI settings for the application -- prompts, parameters, and model-specific logic is entangled in app code.
 
-You can also choose to save outputs to a _different_ file than the original config -- `config.save("history.aiconfig.json", include_outputs=True)`.
+This makes incredibly difficult to iterate on and deploy a production-grade generative AI application. Here are common issues today:
 
-### Why should I use `aiconfig` instead of things like [configurator](https://pypi.org/project/configurator/)?
+- increased complexity to iterate on prompts and model parameters
+- dificulty in switching between models across providers
+- limited collaboration with prompt engineers, who might be domain experts and not developers
+- limited evaluation capabilities for prompt and model performance
 
-It helps to have a [standardized format](http://aiconfig.lastmileai.dev/docs/overview/ai-config-format) specifically for storing generative AI prompts, inference results, model parameters and arbitrary metadata, as opposed to a general-purpose configuration schema.
+AIConfig helps unwind complexity by separating prompts, model parameters, and model-specific logic from your application.
 
-With that standardization, you just need a layer that knows how to serialize/deserialize from that format into whatever the inference endpoints require.
+- simplifies application code -- simply call `config.run()`.
+- open the `aiconfig` in a playground to iterate quickly. prompt engineers can start here and handoff the `aiconfig` to developers.
+- version control and evaluate the `aiconfig` - it's the AI artifact for your application.
 
-### This looks similar to `ipynb` for Jupyter notebooks
+<div style="text-align:center;">
+    <img src="aiconfig-docs/static/img/aiconfig_dataflow.png" alt="AIConfig flow" style="width:40%;">
+</div>
 
-We believe that notebooks are a perfect iteration environment for generative AI -- they are flexible, multi-modal, and collaborative.
+## Get involved
 
-The multi-modality and flexibility offered by notebooks and [`ipynb`](https://ipython.org/ipython-doc/3/notebook/nbformat.html) offers a good interaction model for generative AI. The `aiconfig` file format is extensible like `ipynb`, and AI Workbook editor allows rapid iteration in a notebook-like IDE.
+We are rapidly developing AIConfig! We welcome PR contributors and ideas for how to improve the project.
 
-_AI Workbooks are to AIConfig what Jupyter notebooks are to `ipynb`_
+- [Join the conversation on Discord](https://discord.com/invite/xBhNKTetGx) - `#aiconfig` channel
+- [Open an issue for feature requests](https://github.com/lastmile-ai/aiconfig/issues)
+- [Read our contributing guide](https://aiconfig.lastmileai.dev/docs/contributing/)
```

Comment:
Add a ## Contributing section to make it explicit that we welcome contributions

Reply:
done